app/templates/model-details.html

```diff
@@ -195,11 +191,17 @@ <h2 class="panel-title">Download files</h2>
             <span class="form-control no-fixed-height">Download output movie</span>
           </div>
 
-          <div class="btn-group">
-            <button class="form-control btn btn-primary" id="download-submit" v-on:click="downloadFiles()" value="">
+
+            <button class="btn btn-primary" type="button"  id="download-submit" v-on:click="downloadFiles()" value="">
               Download
             </button>
-          </div>
+
+            <button class="btn btn-primary" type="button" v-bind:class="{ 'inputdisabled': !(model.state === 'SUCCESS' || model.state === 'FAILURE') || isReadOnly() }"  v-on:click="exportModel">
```

Comment:
All buttons should have a type="button" property unless you want the button to be a Submit button:
https://www.w3.org/TR/2011/WD-html5-20110525/the-button-element.html#attr-button-type